### PR TITLE
Enhancement/annotation parsers

### PR DESCRIPTION
**Pull request by jnieuwdorp**
The annotation parsers now fully support the files from blackboard. 
No need to have the ann.csv and imm.csv files from drive. 

Put the Annotation (.gff ) and the resistance conferring mutations (.txt) files from blackboard in your workspace to test.

### CODE REVIEW

File: src/site/checkstyle.xml [hunk 1]
```diff
@@ -89,7 +89,8 @@
         <!-- Checks for imports                              -->
         <!-- See http://checkstyle.sf.net/config_import.html -->
         <module name="AvoidStarImport"/>
-        <module name="IllegalImport"/> <!-- defaults to sun.* packages -->
+        <module name="IllegalImport"/>
+        <!-- defaults to sun.* packages -->
```
Comment (Vennik) [inline src/site/checkstyle.xml]: Are the comments now under the line it's commenting on?

File: src/site/checkstyle.xml [hunk 2]
```diff
@@ -143,7 +146,7 @@
         <module name="IllegalInstantiation"/>
         <module name="InnerAssignment"/>
         <module name="MagicNumber">
-            <property name="ignoreNumbers" value="-1, 0, 1, 2, 3, 4, 5, 100"/>
```
Comment (Vennik) [inline src/site/checkstyle.xml:146]: In my opinion there are now far to many ignores on magic numbers.

Comment (jnieuwdorp) [inline src/site/checkstyle.xml:146]: I need to have the 1 - 10 items from an array for the parser, It's pointless to initiate them on top, I could probably use an index though

edit:did just this in ce95b31

File: src/site/checkstyle.xml (hunk 1 above)
Comment (jnieuwdorp) [inline src/site/checkstyle.xml]: I didn't change this on purpose, I blame IntelliJ

Comment (jnieuwdorp) [inline src/site/checkstyle.xml]: fixed this in d1eaa44, but it was indeed the auto formatting that wants to change it´s location

**Comment by Vennik** [comment]
Looks good now. :shipit:

**Pull request merged**